charts/bitcoind/templates/secrets.yaml

```diff
@@ -0,0 +1,11 @@
+{{ $rpcpassword := include "rpcpassword" . }}
```

Comment:
You will have to change the `rpcpassword` block in `helpers.tpl` to remove the random pass generation. Otherwise this secret and the configmap will end up having a different value for the password.

Reply:
addressed here https://github.com/GaloyMoney/charts/pull/49